explainer.md

```diff
@@ -0,0 +1,66 @@
+
+### Solution
+We propose to provide an API called `getCoalescedEvents` on `PointerEvent` object so that it exposes any events that have been coalesced into the dispatched event.
+This API also matches other platforms such [getHistoricalX/Y](https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int,%20int)) on Android and [similar API](https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_touches_in_your_view/getting_high-fidelity_input_with_coalesced_touches) on iOS.
+There are two options for exposing the coalesced events:
+1. Return a list of dictionary of interesting and useful attributes such as cooridnates.
+2. Return a list of `PointerEvent` object representing the actual events that have been coalesced.
+
+Each item in the result list corresponds to one coalesced event.
+The first option has the benefit of not having an actual `Event` object and hence there is no need to deal with set of functionalities that we don't need them on coaleaced event suchs as propagation functions in the specification.
+The second option gives a more usable way to the developers since if their code is capable of handling a `PointerEvent` object already they could easily just pass the coalesced events to that and it would work fine.
+
+Because of the ease of use for the developers and also after getting some feedback from the developer, the working group chose the second option.
+You can fine examples on how to use the proposed API in the [proposed specification](https://w3c.github.io/pointerevents/extension.html#examples).
+
+
+## Predicted Events
+This is the proposal to match other platforms predicted events API for the events to reduce the general user perception of latency.
+
+### Problem
+Imagine a drawing app that draws the points as it gets from an input device such as mouse.
+There is always some amount of latency compare to when the users draw on a paper due to inherent UA input pipeline latency and graphics pipeline latency and buffering.
+So having some sort of the prediction to draw preemptively a little in the future would reduce the perception of the latency. Having the API as part of the web platform enables UA to use some of the underlying platform APIs as well as using other attributes that might not be exposed to the web directly in the prediction algorithm.
+
+### Solution
+We propose to provide an API called `getPredictedEvents` on `PointerEvent` object. This API format will have the same concerns and API alternatives as `getCoalescedEvent` API introduced earlier.
+So in order to match it with `getCoalescedEvents` as well as exposing an ergonomic API we will use the same format as `getCoalescedEvents` by returning a list of `PointerEvent` objects.
+The events in the result list will have timestamps in the future and UA believes those events are going to be where the pointer is heading.
```

Comment:
"in the future and UA believes those events..." > "in the future, and the coordinates will be the position the UA predicts the pointer is heading to."